src/components/ColorPalette/ColorPalette.js

```diff
@@ -86,31 +103,35 @@ class ColorPalette extends React.PureComponent {
         })}
         style={style}
       >
-        {palette.map((bg, i) => (
-          <button
-            key={i}
-            className="cell-container"
-            onClick={() => {
-              this.setColor(bg);
-            }}
-          >
-            <div
-              className={classNames({
-                'cell-outer': true,
-                active: color?.toHexString()?.toLowerCase() === bg?.toLowerCase(),
-              })}
+        {palette.map(bg => bg?.toLowerCase()).map((bg, i) => (
+          !bg 
+          ? <div key={i} className="dummy-cell" /> 
+          : <button
+              key={i}
+              className="cell-container"
+              onClick={() => {
+                this.setColor(bg === 'transparency' ? null : bg);
+              }}
             >
               <div
                 className={classNames({
-                  cell: true,
-                  border: bg.toLowerCase() === '#ffffff',
+                  'cell-outer': true,
```

Comment:
Since we are adding special checking for `transparency`, it looks like the code starting from line 134 is not needed anymore? To maintain the original behavior we can just add a `transparency` to the end of `this.palette` array.

Reply:
Thanks for pointing this out. I made this change